src/cgroup.rs

```diff
@@ -59,43 +59,76 @@ impl Default for Cgroup {
 }
 
 impl Cgroup {
+    pub fn v2(&self) -> bool {
+        self.hier.v2()
+    }
+
     /// Create this control group.
-    fn create(&self) {
+    fn create(&self) -> Result<()> {
         if self.hier.v2() {
-            let _ret = create_v2_cgroup(self.hier.root(), &self.path);
+            create_v2_cgroup(self.hier.root(), &self.path, None)
         } else {
             for subsystem in &self.subsystems {
                 subsystem.to_controller().create();
             }
+            Ok(())
         }
     }
 
-    pub fn v2(&self) -> bool {
-        self.hier.v2()
+    /// Create this control group with specified controllers.
+    fn create_with_specified_controllers(
+        &self,
+        specified_controllers: Option<Vec<String>>,
```

Comment:
The variable `specified_controllers` seems only work with Cgroup V2?
I wonder:

- Will this function be called when using Cgroup v1?
- If yes, what will happen if `specified_controllers` do not match with `self.subsystems`?
- If no, Why we do not make judgments in new and load?

Reply:
Yes, cgroup v1 also requires filtering on specified controllers. I modified to filter in new and load.